### ISSUE TITLE
SVHN validation dataset problem

### ISSUE THREAD

**Issue by itayhubara** [nicholas-leonard/dp#184]
Hi,

I am trying to use the code for SVHN normalization (GCN+LCN) and save the results.   I receive that the validation dataset (i.e ds:validSet():inputs():input()) is the same size as the training dataset. Moreover, while the training and test set converge nicely the validation set receive ridicules results.
I believe their is a bug...
Can you help?
Best
Itay
